### ISSUE TITLE
[r29] Exported Universal Windows App cant be opened with VS2017 & the fix 

### ISSUE THREAD

**Issue by JeFawk** [Scirra/Construct-3-bugs#253]
## Problem description
Issue discovered by **_saiyadjin_**
My explanation: when exporting as a universal windows app, the export cannot be opened with VS2017 which states that it's incompatible. There are differences in what VS2017 creates as a blank JS app and what is exported in the .jsproj file. The original .jsproj created by VS2017 contains the modified version number of 15 instead of 14 what is exported as well as a PackageCertificateKeyFile. VS2017 also creates the certificate file.

## Steps to reproduce

1. Export any template as a universal app
2. Try to open the solution with VS2017 community edition

## Observed result

It will show as incompatible.
![image](https://cloud.githubusercontent.com/assets/7268697/26219030/ef1e8460-3c15-11e7-9bbb-ae4a1b4e8b0e.png)

I've attached 2 exports of Ghost shooter. One that doesn't work, and one that works with modifications I made to the project. Notice the certificate and the differences in the .jsproj  file
I didnt have time to test if the certificate was **really** needed.

![image](https://cloud.githubusercontent.com/assets/7268697/26219192/86cfbc20-3c16-11e7-800d-f9d812fc51e1.png)

[not working - Ghost_Shooter.zip](https://github.com/Scirra/Construct-3-bugs/files/1012049/not.working.-.Ghost_Shooter.zip)
[working - Ghost shooter.zip](https://github.com/Scirra/Construct-3-bugs/files/1012050/working.-.Ghost.shooter.zip)

**Comment by AshleyScirra** [comment]
You're right, VS2017 can't seem to open the project C3 exports. I'm not sure why this is, I would have thought VS would be backwards-compatible.

This is difficult to solve because:
- ideally we can just export one project that is forwards-compatible rather than having to make the user choose & support different VS project formats
- it's not clear how to generate a valid .pfx file in the correct format (and securely), I'd rather C3 didn't have to do that

I've got in touch with a contact at Microsoft to see what they say about this.

**Comment by AshleyScirra** [comment]
It turns out VS2017 fails to load the project if the Windows 10 SDK version the VS project uses isn't installed. You can actually directly open the VS project C3 exports, providing the Windows 10 SDK version set in `<TargetPlatformVersion>` in the .jsproj is installed. I think C2 defaults this to 10.0.10240.0, but I verified that using 10.0.14393.0 with that SDK installed also allowed the project to be opened in VS2017.

This is a bit awkward since there doesn't seem to be an "auto" setting for that field and I'd prefer not to have to make the user enter it, but that's probably the best solution at least for the short term.

**Comment by AshleyScirra** [comment]
After talking to Microsoft, think the best solution is just to prompt the user for the Windows 10 SDK version. You only need to enter the SDK version once then your Windows 10 exports always work, and can open in both VS2015 and VS2017 (with the extra bonus VS2015 works without having to tweak project settings first). This is added for r31. Verified that it allows direct export and opening with VS2017.
